app/src/forms/form/encryptionKey/routes.js

```diff
@@ -0,0 +1,22 @@
+const routes = require('express').Router();
+const { currentUser, hasFormPermissions } = require('../../auth/middleware/userAccess');
+const validateParameter = require('../../common/middleware/validateParameter');
+const apiAccess = require('../../auth/middleware/apiAccess');
+const P = require('../../common/constants').Permissions;
+
+const controller = require('./controller');
+
+routes.use(currentUser);
+
+routes.param('formId', validateParameter.validateFormId);
+routes.param('formEncryptionKeyId', validateParameter.validateFormEncryptionKeyId);
+
+routes.get('/encryptionKey/algorithms', apiAccess, async (req, res, next) => {
+  await controller.listEncryptionAlgorithms(req, res, next);
+});
+
+routes.get('/:formId/encryptionKey/:formEncryptionKeyId', apiAccess, hasFormPermissions([P.FORM_READ, P.FORM_UPDATE]), async (req, res, next) => {
```

Comment:
Just curious if there is a use case for retrieving the form's encryption key using the api key. Or was this possibly copied over and isn't something we want to offer?

Reply:
ah, sorry. understand your question. i'll remove API access

Reply:
The API docs should be updated too, to remove the BasicAuth.

Reply:
yup, got it.